packages/blog/src/blog-fns.ts

```diff
@@ -4,9 +4,9 @@ import parseFrontMatter from "./parseFrontMatter";
 import { slugify } from "./slugify";
 
 type Cache = {
```

Comment:
Hmm.. Cache issues spring to mind when considering that getPosts has Options, and that those Options aren't considered?

Reply:
I guess it's ok for the Cache to hold everything and the fitlering/limits to operate over that. Not sure how to handle typing the cache if it's defined like this though.